src/vm/runners/cairo_runner.rs

```diff
@@ -516,20 +517,26 @@ impl CairoRunner {
     pub fn run_until_pc(
         &mut self,
         address: Relocatable,
+        run_resources: &mut Option<RunResources>,
```

Comment:
Maybe this could be `Option<&mut RunResources>` instead?
As to avoid having to call it with `&mut None`

Reply:
i was unsure which one would be a better option but the suggestion is fair enough, besides it matches the behavior it has in the function.

Reply:
It will be ugly but I think `mut Option<&mut RunResources>` is likely to work.